### DISCUSSION TITLE
设置Default service provider里的ChatGPT(Web)是什么原理？选这个不需要输入api就能免费使用chatgpt3.5？

### DISCUSSION THREAD

**Discussion by GoTop** [nextai-translator/nextai-translator#1131] [Q&A]
设置Default service provider里的ChatGPT(Web)是什么原理？选这个不需要输入api就能免费使用chatgpt3.5？

**Comment by ainlee** [comment]
同問～而且為什麼本地安裝的版本找不到這個項目了？同理是不是也可以加入Copilot呢？

**Comment by fengdi721** [comment]
同问，为什么有的时候完全没有反应，有的时候可以正常使用

**Reply by ainlee** [reply]
我發現如果你沒有登入ChatGPT的網頁，他就不會運作....你可以試試
